modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/capability/xml/TestAnotherDocumentedConfig.java

```diff
@@ -14,7 +14,7 @@
 /**
  * This is some Another Config documentation.
  */
-@Configuration(name = "anotherConfig")
+@Configuration(name = "another")
```

Comment:
I don't understand how this is a fix. ConfigLoaderDelegate only appends the -config if the name doesn't already end in config